t

```diff
@@ -0,0 +1,31 @@
+[1mdiff --git a/src/vivarium_census_prl_synth_pop/constants/data_values.py b/src/vivarium_census_prl_synth_pop/constants/data_values.py[m
```

Comment:
Delete this file. I bet you have an alias that got jumbled since this is exactly the same thing that happened in your last PR (a diff is generated and saved to a file called `t`)